MapboxDirections/MBRouteOptions.swift

```diff
@@ -181,6 +181,13 @@ open class RouteOptions: NSObject, NSSecureCoding {
         self.attributeOptions = attributeOptions
         
         includesExitRoundaboutManeuver = decoder.decodeBool(forKey: "includesExitRoundaboutManeuver")
+        
+        
+        guard let excludeClassesDescriptions = decoder.decodeObject(of: NSString.self, forKey: "excludeRoadClasses") as String?,
+            let excludeRoadClasses = RoadClasses(descriptions: excludeClassesDescriptions.components(separatedBy: ",")) else {
+                return nil
```

Comment:
By returning `nil` here, we’re rejecting any encoded RouteOptions object that lacks an `excludedRoadClasses`, even one that was created before this PR lands. Let’s quietly set `excludedRoadClasses` to `nil` instead.

Reply:
```swift
let excludedRoadClassesDescriptions = decoder.decodeObject(of: NSString.self, forKey: "excludedRoadClasses") as String?
excludedRoadClasses = RoadClasses(descriptions: excludedRoadClassesDescriptions?.components(separatedBy: ",") ?? "")
```